roles/migrationcontroller/templates/controller_config.yml.j2

```diff
@@ -25,6 +25,7 @@ data:
   STUNNEL_POD_MEMORY_LIMIT: "{{ stunnel_pod_memory_limits }}"
   STUNNEL_POD_CPU_REQUEST: "{{ stunnel_pod_cpu_requests }}"
   STUNNEL_POD_MEMORY_REQUEST: "{{ stunnel_pod_memory_requests }}"
+  RSYNCD_ROUTE_SELECTOR: "{{ migration_rsyncd_route_selector | to_json }}"
```

Comment:
is there some validation that we can do here to make sure that this does not blow up the controller?